bigquery/google/cloud/bigquery/magics.py

```diff
@@ -39,6 +39,9 @@
         amount of time for the query to complete will not be cleared after the
         query is finished. By default, this information will be displayed but
         will be cleared after the query is finished.
+    * ``--params <params dictionary>`` (optional, line argument):
+        If present, the argument will be parsed into a dictionary. This dictionary
+        will be used to format values enclosed within {} in the query.
```

Comment:
The docs here need to spell out the format of the value (e.g., as done below, "a parseable JSON string").